src/Connection.js

```diff
@@ -0,0 +1,52 @@
+
```

Comment:
Try to get a better code structure + naming
/Communication/XComponentConnection